index.d.ts

```diff
@@ -0,0 +1,4 @@
+// TypeScript Version: 3.5.2
+
+export declare function createWriter<T>(redis: T, key: string): T;
+export declare function createReader<T>(redis: T, key: string, opt?: T): T;
```

Comment:
*If* that's reasonably easy in TypeScript definition files, please specify that it returns a [Node `Readable` stream](https://nodejs.org/api/stream.html#stream_readable_streams).